src/applications/vaos/utils/eligibility.js

```diff
@@ -15,20 +39,26 @@ export async function getEligibilityData(
 ) {
   const facilityId = facility.institutionCode;
   const eligibilityChecks = [
-    checkPastVisits(systemId, facilityId, typeOfCareId, 'request'),
-    getRequestLimits(facilityId, typeOfCareId),
+    checkPastVisits(systemId, facilityId, typeOfCareId, 'request').catch(
```

Comment:
Adding these catch statements to everything so that Promise.all never fails and we can do our own error logic after all the calls finish.